src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +72,13 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * Should be overwritten in child class
+     * @return
+     */
+    public boolean isMutating() {
```

Comment:
Is it okay to have the default behavior of any command to be always mutating?  Maybe we should write the code in such a way that this method is always overridden in its subclasses